.github/workflows/python_tools.yml

```diff
@@ -51,6 +51,18 @@ jobs:
       run: |
        python -m unittest discover -p '*_test.py' --failfast -v
       working-directory: ./tools/explorer
+    #---------- Configuration scoring ----------#
```

Comment:
undo? keep the scorer after the validator since there is a dependency

Reply:
Ah ok, this is incorrect in g3 as well then. Will remove.

Reply:
did you remove? I am not sure if I am looking at the latest version?